cmake/compiler/clang/target_arm.cmake

```diff
@@ -25,6 +25,9 @@ if(CONFIG_FPU)
   elseif(CONFIG_FP_SOFTABI)
     list(APPEND ARM_C_FLAGS   -mfloat-abi=softfp)
   endif()
+else()
+  list(APPEND ARM_C_FLAGS   -mfpu=none)
+  list(APPEND ARM_C_FLAGS   -mfloat-abi=soft)
```

Comment:
Could you add a comment that `-mfloat-abi=soft` disables use of FPU registers (it uses General Purpose Registers instead)?

Reply:
Done.